lineman/app/components/group_page/membership_request_form/membership_request_form.haml

```diff
@@ -0,0 +1,24 @@
+%form.membership-request-form{ng-submit: 'submit()', ng-disabled: 'membership_request.processing'}
+  .modal-header
+    %button{type: 'button', class: 'close', ng-click: '$close()'}
+      %span{aria-hidden: true} &times;
+      %span.sr-only{translate: 'common.action.cancel'}
+    %h1.lmo-h1{translate: 'membership_request_form.heading'}
+
+  .modal-body
+    %fieldset
+      .form-group
+        %label{for: 'membership-request-name', translate: 'membership_request_form.name_label'}
+        %input.membership-request-form__name#membership-request-name.form-control{ng-model: 'membershipRequest.actor().name', ng-required: 'true', ng-disabled: 'true'}
```

Comment:
Ooh, you could give a hoon to the new %validation_errors directive once that gets merged in here.